sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/delete_key_operation.hpp

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief A long-running operation for deleting a Key.
+ *
+ */
+
+#pragma once
+
+#include <azure/core/http/http.hpp>
+#include <azure/core/operation.hpp>
+#include <azure/core/operation_status.hpp>
+#include <azure/core/response.hpp>
+
+#include <azure/keyvault/common/internal/keyvault_pipeline.hpp>
+#include <azure/keyvault/common/keyvault_exception.hpp>
+
+#include "azure/keyvault/keys/deleted_key.hpp"
```

Comment:
Why do we drop the Security folder from the on disk path?
i.e. `#include "azure/security/keyvault/keys/...."`

Reply:
I thought it was easier to include like that, since we keep the azure/service-name/package-name without adding an extra grouping concept for the header inclusion.
I was leaving only the NameSpace for the code to mention about the code grouping.

Do we need to also add the grouping name to the include path?  If yes, I can create a new issue to update all the current includes from KeyVault

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/1692